config/initializers/google_drive_vars.rb

```diff
@@ -12,3 +12,4 @@
 
 DSI_USER_SPREADSHEET_ID = ENV['DSI_USER_SPREADSHEET_ID']
 DSI_USER_WORKSHEET_GID = ENV['DSI_USER_WORKSHEET_GID']
+DSI_APPROVER_WORKSHEET_GID = ENV['DSI_APPROVER_WORKSHEET_GID']
```

Comment:
This will require some terraform changes.

Reply:
Yep, the terraform changes will be made in the next PR